scripts/create_empty_scalars.py

```diff
@@ -82,41 +101,57 @@ def expand_scalars(df, column, where, expand):
     return _df_wo_cc
 
 
+def add_wacc(sc):
```

Comment:
This function is very specific. Why not instead write a function that adds a line to the scalars and then pass the specific data to it?

Reply:
Good idea! Thanks. Done something like this with commit https://github.com/rl-institut/oemof-B3/pull/307/commits/00606929469bd38bd30b0aa9fd660bf3e7085055.